core/platform/models_test.py

```diff
@@ -0,0 +1,271 @@
+            self.registry_instance.import_models([models.NAMES.email]))
+
+    def test_import_models_exploration(self):
+        """Tests import_models function with exploration option."""
+        from core.storage.exploration import gae_models as exp_models
+        expected_exploration_models = (exp_models,)
+        self.assertEqual(
+            expected_exploration_models,
+            self.registry_instance.import_models([models.NAMES.exploration]))
+
+    def test_import_models_feedback(self):
+        """Tests import_models function with feedback option."""
+        from core.storage.feedback import gae_models as feedback_models
+        expected_feedback_models = (feedback_models,)
+        self.assertEqual(
+            expected_feedback_models,
+            self.registry_instance.import_models([models.NAMES.feedback]))
+
+    def test_import_models_file(self):
+        """Tests import_models function with file option."""
+        from core.storage.file import gae_models as file_models
+        expected_file_models = (file_models,)
+        self.assertEqual(
+            expected_file_models,
+            self.registry_instance.import_models([models.NAMES.file]))
+
+    def test_import_models_job(self):
+        """Tests import_models function with job option."""
+        from core.storage.job import gae_models as job_models
+        expected_job_models = (job_models,)
+        self.assertEqual(
+            expected_job_models,
+            self.registry_instance.import_models([models.NAMES.job]))
+
+    def test_import_models_question(self):
+        """Tests import_models function with question option."""
+        from core.storage.question import gae_models as question_models
+        expected_question_models = (question_models,)
+        self.assertEqual(
+            expected_question_models,
+            self.registry_instance.import_models([models.NAMES.question]))
+
+    def test_import_models_recommendations(self):
+        """Tests import_models function with recommendations option."""
+        from core.storage.recommendations import gae_models as recommendations_models # pylint: disable=line-too-long
+        expected_recommendations_models = (recommendations_models,)
+        self.assertEqual(
+            expected_recommendations_models,
+            self.registry_instance.import_models(
+                [models.NAMES.recommendations]))
+
+    def test_import_models_skill(self):
+        """Tests import_models function with skill option."""
+        from core.storage.skill import gae_models as skill_models
+        expected_skills_models = (skill_models,)
+        self.assertEqual(
+            expected_skills_models,
+            self.registry_instance.import_models([models.NAMES.skill]))
+
+    def test_import_models_statistics(self):
+        """Tests import_models function with statistics option."""
+        from core.storage.statistics import gae_models as statistics_models # pylint: disable=line-too-long
```

Comment:
Is the pylint pragma needed? (Line 150 looks like it's around the same length and doesn't need one.)

Reply:
Removed!